sentry-core/src/main/java/io/sentry/core/EnvelopeSender.java

```diff
@@ -1,5 +1,6 @@
 package io.sentry.core;
 
+import io.sentry.core.hints.*;
```

Comment:
The last few commits were always with wildcards, not good, I'd fix it.

Reply:
I've tried to change this multiple times.
Based on this:
https://intellij-support.jetbrains.com/hc/en-us/community/posts/206203659-Turn-off-Wildcard-imports-

It's here:

<img width="301" alt="image" src="https://user-images.githubusercontent.com/1633368/68890070-30438000-06ec-11ea-9fbc-0e78cdbb15b6.png">
It's always adding it now though

Reply:
I usually set these to 999 to avoid star imports

Reply:
It's adding it for a single imports for me. And no change to that setting makes any difference.
Nothing on `.editorconfig`, no idea what's going on

Reply:
I think we should force it on `.editorconfig`, maybe nothing means default value which is just a few start imports